OmniKit/MessageTransport/MessageBlocks/PodInfo.swift

```diff
@@ -18,7 +18,7 @@ public protocol PodInfo {
 public enum PodInfoResponseSubType: UInt8, Equatable {
     case normal                      = 0x00
     case configuredAlerts            = 0x01
-    case faultEvents                 = 0x02
+    case detailedStatus              = 0x02
```

Comment:
Probably worth adding a comment here or elsewhere that mentions that the detailedStatus is returned on any pod fault.